src/utils/generate-rest-contract.ts

```diff
@@ -0,0 +1,10 @@
+import { RestContract } from '../types';
+
+export function generateRestContract(swaggerFile: object): RestContract {
```

Comment:
it isn't always a swaggerFile that's being passed to generateRestContract, can you rename the fn param?